specification/network/resource-manager/Microsoft.Network/stable/2024-05-01/virtualWan.json

```diff
@@ -1376,7 +1376,57 @@
         }
```

Comment:
add examples for all new APIs. update readMe file to introduce this in a new API version